docs/developers/tutorials/staking-contract.md

```diff
@@ -247,22 +241,16 @@ You have to press "space" between the words, not "enter"!
 
 [comment]: # (mx-context-auto)
 
-### Deploying the contract
+### Deploying the contract on devnet
 
-Now that we've created a wallet, it's time to deploy our contract. Open your `snippets.sh` file, and add the following:
+Now that we've created a wallet, it's time to deploy our contract, make sure you build the contract before deploying it. Open a command line and run the following command:
 
 ```bash
-USER_PEM="~/Downloads/tutorialKey.pem"
-PROXY="https://devnet-gateway.multiversx.com"
-CHAIN_ID="D"
-
-deploy() {
-    mxpy --verbose contract deploy --project=${PROJECT} \
-    --recall-nonce --pem=${USER_PEM} \
+mxpy --verbose contract deploy --project=/Projects/staking-contract/ \
+    --recall-nonce --pem=/Downloads/tutorialKey.pem \
     --gas-limit=10000000 \
     --send --outfile="deploy-devnet.interaction.json" \
-    --proxy=${PROXY} --chain=${CHAIN_ID} || return
-}
+    --proxy=https://devnet-gateway.multiversx.com --chain=D || return
```

Comment:
Can be without `|| return` in this context.

Reply:
After the command snippet, we can also state:

_Upon running this command, the address of the deployed contract is displayed in console. Make sure to note it down / save it / retain it_.

Reply:
it is already stated that the address of the contract it is displayed in console